ctk/common/src/main/java/org/codice/compliance/utils/sign/SimpleSign.java

```diff
@@ -56,21 +58,20 @@
 import org.opensaml.xmlsec.signature.support.SignatureConstants;
 import org.opensaml.xmlsec.signature.support.SignatureValidator;
 import org.opensaml.xmlsec.signature.support.provider.ApacheSantuarioSignatureValidationProviderImpl;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class SimpleSign {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleSign.class);
-
   private final SystemCrypto crypto;
 
-  private static final Map<String, String> URI_ALG_MAP = new HashMap<>();
-
-  static {
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#dsa-sha1", "SHA1withDSA");
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#rsa-sha1", "SHA1withRSA");
-  }
+  private static final ImmutableMap<String, String> URI_ALG_MAP =
```

Comment:
❗️ We should get rid of this custom map in place of just using https://santuario.apache.org/Java/api/org/apache/xml/security/algorithms/JCEMapper.html. You can call `JCEMapper.translateURItoJCEID(sigAlg);` where `sigAlg` is the URI format and it'll spit out either `null` or the JCE format i.e. "SHA1withDSA".